src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -1737,3 +1737,43 @@ def test_managed_private_endpoints(self):
         self.cmd(
             'az synapse managed-private-endpoints show --workspace-name {workspace} --pe-name {name}',
             expect_failure=True)
+
+    @record_only()
```

Comment:
May I know why it's record only?

Reply:
for some tests, we rely on pre-existing resources and records: eg. sparkpool, workspaces ... That's why we use record_only

Reply:
Can we add test case without relying on pre-existing resources?